vault/resource_pki_secret_backend_role.go

```diff
@@ -28,48 +28,49 @@ var (
 // Any new fields should probably not be added to these lists. Instead handle
 // them separately within a provider.IsAPISupported guard
 var pkiSecretFields = []string{
-	consts.FieldTTL,
-	consts.FieldMaxTTL,
 	consts.FieldAllowedDomainsTemplate,
-	consts.FieldAllowedURISans,
 	consts.FieldAllowedOtherSans,
-	consts.FieldServerFlag,
+	consts.FieldAllowedURISans,
 	consts.FieldClientFlag,
 	consts.FieldCodeSigningFlag,
+	consts.FieldCountry,
 	consts.FieldEmailProtectionFlag,
-	consts.FieldKeyType,
 	consts.FieldKeyBits,
+	consts.FieldKeyType,
+	consts.FieldLocality,
+	consts.FieldMaxTTL,
+	consts.FieldNotBeforeDuration,
 	consts.FieldOU,
 	consts.FieldOrganization,
-	consts.FieldCountry,
-	consts.FieldLocality,
+	consts.FieldPostalCode,
 	consts.FieldProvince,
+	consts.FieldServerFlag,
 	consts.FieldStreetAddress,
-	consts.FieldPostalCode,
-	consts.FieldNotBeforeDuration,
+	consts.FieldTTL,
 }
 
 var pkiSecretListFields = []string{
 	consts.FieldAllowedDomains,
-	consts.FieldKeyUsage,
-	consts.FieldExtKeyUsage,
 	consts.FieldAllowedSerialNumbers,
+	consts.FieldExtKeyUsage,
+	consts.FieldKeyUsage,
 }
 
 var pkiSecretBooleanFields = []string{
-	consts.FieldAllowLocalhost,
+	consts.FieldAllowAnyName,
 	consts.FieldAllowBareDomains,
-	consts.FieldAllowSubdomains,
 	consts.FieldAllowGlobDomains,
-	consts.FieldAllowAnyName,
-	consts.FieldEnforceHostnames,
 	consts.FieldAllowIPSans,
-	consts.FieldUseCSRCommonName,
-	consts.FieldUseCSRSans,
+	consts.FieldAllowLocalhost,
+	consts.FieldAllowSubdomains,
+	consts.FieldAllowedURISansTemplate,
```

Comment:
This is the new field `allowed_uri_sans_template`. There were no other additions/deletions. All other fields were sorted